### PR TITLE
Add NexusPIPE / cdn

### PR DESCRIPTION
**Pull request by Jack-Bailey**

### CODE REVIEW

**Comment by Jack-Bailey** [comment]
Original issue: #6547

File: src/technologies/n.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
 {
+  "NexusPIPE": {
+    "cats": [
+      31
+    ],
+    "description": "NexusPIPE is a ADC & DDoS mitigation Company.",
+    "headers": {
+      "Server": "^Nexuspipe.com"
+    },
+    "icon": "NexusPIPE.png",
```
Comment (nurbek91) [inline src/technologies/n.json:10]: 32x32
NexusPIPE

File: src/technologies/n.json [hunk 2]
```diff
@@ -1,4 +1,15 @@
 {
+  "NexusPIPE": {
+    "cats": [
+      31
```
Comment (nurbek91) [inline src/technologies/n.json:4]: We can add a  Security category 16

File: src/technologies/n.json [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
+    ],
+    "description": "NexusPIPE is a ADC & DDoS mitigation Company.",
```
Comment (nurbek91) [inline src/technologies/n.json:6]: NexusPIPE is a ADC and DDoS mitigation company.

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate thanks for PR!!!
need small changes

**Comment by Jack-Bailey** [comment]
No problem - Added your changes

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**